server/routes/routerRequestHandlers/retrieveCuriousFunctionalSkills.ts

```diff
@@ -5,12 +5,17 @@ import { Result } from '../../utils/result/result'
 /**
  *  Middleware function that returns a Request handler function to look up the prisoner's functional skills from Curious
  */
-const retrieveCuriousFunctionalSkills = (curiousService: CuriousService): RequestHandler => {
+const retrieveCuriousFunctionalSkills = (
+  curiousService: CuriousService,
+  options: { useCurious1ApiForFunctionalSkills: boolean } = { useCurious1ApiForFunctionalSkills: false },
+): RequestHandler => {
   return async (req: Request, res: Response, next: NextFunction) => {
     const { prisonNumber } = req.params
 
     // Lookup the prisoners functional skills and store in res.locals
-    res.locals.prisonerFunctionalSkills = await Result.wrap(curiousService.getPrisonerFunctionalSkills(prisonNumber))
+    res.locals.prisonerFunctionalSkills = await Result.wrap(
```

Comment:
This is the change to the middleware that I mentioned earlier .... it now takes an optional option object which is simply passed straight down into the service method (see below 👇 )